install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -61,6 +61,12 @@ etcd3.example.com
 $ ansible-playbook [-i /path/to/file] \
   /usr/share/ansible/openshift-ansible/playbooks/openshift-etcd/scaleup.yml
 ----
++
+.. Set the node label to `logging-infra-fluentd: "true"`.
++
```

Comment:
@bmcelvee Should this be Step 3, rather than 2i? Seems odd having only one sub-step.

Reply:
Sure! Updated. :maple_leaf: Thanks!